python/nav/django/settings.py

```diff
@@ -217,6 +217,8 @@
     'django.contrib.staticfiles',
     'django.contrib.sessions',
     'django.contrib.humanize',
+    'django.contrib.contenttypes',
+    'django.contrib.auth',
```

Comment:
This will drag in the contentypes-tables and the Django auth models. Do we want that?

If so, we'd need the sql for these as NAV is older than Django's own migration system and doesn't use it.

Reply:
We don't want it, but if `drf-oidc-auth` requires it for some strange reason, we might have to. I mentioned this in my comment before my review.

Reply:
Manage to get rid of this in the fork